src/features/user/components/profile.tsx

```diff
@@ -0,0 +1,41 @@
+import { useMutation } from "@tanstack/react-query"
+
+import { updateImage } from "../utils/fetch-data"
+import { useAuth } from "../../../shared/hooks/useAuth"
+
+
+
+const img = "https://img.freepik.com/premium-photo/chita_827316-164.jpg"
+const border = "border-[#41c48b] border-[3px]"
```

Comment:
Niby fajnie, że nie chcesz powtarzać kodu, ale to jest tailwind tutaj tego aż tak przestrzegać nie musisz. Wyjeb tą zmienną i po prostu użyj tych styli bez zmiennej